### PR TITLE
SessionList and SessionListItem components

### PR DESCRIPTION
**Pull request by SOLEIGH**
adding the SessionList and SessionListItem components

### CODE REVIEW

File: src/components/SessionListItem/SessionListItem.module.scss [hunk 1]
```diff
@@ -0,0 +1,35 @@
+@import "../../styles/abstracts/_mixins.scss";
+@import "../../styles/abstracts/_variables.scss";
+
+.wrapper{
+    @include component;
+  
+    display: flex;
+    justify-content: space-between;
+    margin: 10px;
```
Comment (albonkey) [inline src/components/SessionListItem/SessionListItem.module.scss]: This margin makes the list item not align with the header. Can be removed, and rather use `gap: 10px` in the SessionList.module.scss file.

File: src/components/SessionListItem/SessionListItem.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+import style from './SessionListItem.module.scss';
+
+const SessionListItem = ({session}) => {
+    const [isSelected, setIsSelected] = useState(false);
```
Comment (albonkey) [inline src/components/SessionListItem/SessionListItem.js]: The isSelected doesnt really do anything here, so no need to include it

File: src/components/SessionListItem/SessionListItem.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+                <span className={style.status}>
+                    {
+                        session.isTeaching ? 'Teaching' : 'Learning'
+                    }
+                </span>
+                <span>{`${session.subject} with ${session.name}`}</span>
+            </div>
+            <div className={style.info}>
+                <a className={style.link} 
+                    onClick={() => setIsSelected(!isSelected)}
```
Comment (albonkey) [inline src/components/SessionListItem/SessionListItem.js]: Same for this onclick part here, can be removed, but keep the link just not the onClick part

File: src/components/SessionList/SessionList.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import style from './SessionList.module.scss';
+import SessionListItem from '../SessionListItem/SessionListItem';
+
+const SessionList = ({title, sessions}) => {
+    return(
+        <div className = {style.wrapper}>
+			<div className= {style.heading2}>
```
Comment (albonkey) [inline src/components/SessionList/SessionList.js:8]: The code should have different spacing, make sure to only use one tab when nesting divs

**Review by albonkey — CHANGES_REQUESTED** [review]
Left a couple comments on things that can be fixed up :)

File: src/components/SessionList/SessionList.js (hunk 4 above)
Comment (SOLEIGH) [reply src/components/SessionList/SessionList.js:8]: I see the spacing issue here, but in the editor it is only one tab for each div. ???

**Review by albonkey — APPROVED** [review]
Looks good, you can go ahead and merge the changes :)

**Pull request merged**